example/lib/main.dart

```diff
@@ -70,6 +75,10 @@ class _DittoExampleState extends State<DittoExample> {
       token: token,
     );
 
+    //
+    // Uncomment the following lines to use authentication with Ditto.
+    //
+
     // final identity = await OnlineWithAuthenticationIdentity.create(
```

Comment:
Oops, this commented-out snippet is ancient (pre-GA). Nothing wrong with your PR, but we should remove `await` and `.create` - there's a regular synchronous constructor now